src/blanket_traits.rs

```diff
@@ -17,6 +17,11 @@ use core::{fmt, hash};
 
 use crate::MiniscriptKey;
 
+/// Auxiliary trait indicating that a type implements both `Debug` and `Display`.
```

Comment:
```suggestion
/// Auxiliary trait indicating that a type implements `'static`, `Debug` and `Display`.
```

Reply:
"implements `'static`" isn't the right terminology ... if we want to say this would say it "is `'static`".

But IMHO it's fine as-is. The name of the trait exactly describes what it does.

Reply:
Works for me.